app/views/cookies/cookie-policy.njk

```diff
@@ -17,7 +17,7 @@
 
   {% if updated %}
     {% set html %}
-      <p class="govuk-notification-banner__heading">
+    <p class="govuk-notification-banner__heading">
```

Comment:
check indentation please.

Reply:
This indentation is set by Javascript standard, if I overwrite it and indent, the spacing is removed when I save.